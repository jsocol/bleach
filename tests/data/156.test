<div id="86"><body oninput=alert(86)><input autofocus>//["'`-->]]>]</div><div id="87"><svg xmlns="http://www.w3.org/2000/svg">
<a xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="javascript:alert(87)"><rect width="1000" height="1000" fill="white"/></a>
</svg>//["'`-->]]>]</div>
