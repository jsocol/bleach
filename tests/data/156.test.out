<div id="86"><input>//["'`--&gt;]]&gt;]</div><div id="87"><svg xmlns="http://www.w3.org/2000/svg">
<a><rect fill="white" height="1000" width="1000"></rect></a>
</svg>//["'`--&gt;]]&gt;]</div>
